services/editor/src/pages/settings/components/HooksPage/WebHookFormats.js

```diff
@@ -0,0 +1,9 @@
+export const webhookFormats = [
+  { label: 'JSON', value: 'json' },
+  { label: 'Slack', value: 'slack' },
+];
+
+export const webhookLabelsByFormat = webhookFormats.reduce((labelsByFormat, formatObject) => {
+  labelsByFormat[formatObject.value] = formatObject.label;
+  return labelsByFormat;
+}, {});
```

Comment:
maybe: (to preserve reduce pure semantics)
```suggestion
export const webhookLabelsByFormat = webhookFormats.reduce((labelsByFormat, formatObject) => ({
  ...labelsByFormat,
  [formatObject.value]:formatObject.label
}), {});
```

Reply:
Yeah I agree, It's copy-pasted, Ill fix if where I copied it from

Reply:
fixed